packages/tooltip/src/index.js

```diff
@@ -232,7 +232,6 @@ export function useTooltip({
   onMouseLeave,
   onFocus,
   onBlur,
-  onKeyDown,
```

Comment:
We need to keep this, and any event handler we use internally should be passed as the second argument in our `wrapEvent` function. This allows developers who use our components to stop our event handlers when they call `event.preventDefault`.